conf/quicklogic/environment.yml

```diff
@@ -6,6 +6,9 @@ dependencies:
   - litex-hub::quicklogic-yosys=v0.8.0_45_gcb3eef2b=20201216_065815
   - litex-hub::quicklogic-yosys-plugins=v1.2.0_11_g21045a9=20201216_065815
   - litex-hub::quicklogic-vtr=v8.0.0.rc2_4003_g8980e4621=20200902_114536
+  # TODO: remove when quicklogic-vtr will be updated to version built after: https://github.com/hdl/conda-eda/pull/90
+  # or when conda-forge will rename API breaking tbb to another name: https://github.com/conda-forge/tbb-feedstock/issues/81
+  - tbb=2020.2
```

Comment:
@kamilrakoczy something weird has happened in conda: 

```
'name: quicklogic   # or when conda-forge will rename API breaking tbb to another name: https://github.com/conda-forge/tbb-feedstock/issues/81' 'quicklogic # or when conda-forge will rename API breaking tbb to another https://github.com/conda-forge/tbb-feedstock/issues/81'
third_party/make-env/conda.mk:86: *** "Space not allowed in conda environment name: ' ' 'quicklogic # or when conda-forge will rename API breaking tbb to another https://github.com/conda-forge/tbb-feedstock/issues/81' 'quicklogic?#?or?when?conda-forge?will?rename?API?breaking?tbb?to?another?https://github.com/conda-forge/tbb-feedstock/issues/81'".  Stop.
```

I believe that the `name: https://...` in the comment above messes up with the yml interpreter which believes that also part of that comment belongs to the environment name.

Reply:
Ok, I changed ``name:`` to ``name`` do not confuse yml interpreter.